vmdb/app/models/container_node.rb

```diff
@@ -8,4 +8,10 @@ class ContainerNode < ActiveRecord::Base
   belongs_to :lives_on, :polymorphic => true
 
   delegate   :hardware, :to => :computer_system
+
+  virtual_column :condition_ready, :type => :string, :uses => :container_node_conditions
+
+  def condition_ready
+    container_node_conditions.find_by_name('Ready').status
```

Comment:
@dkorn - this is a Rails model with a `has_many` association called `container_node_conditions`.  So, there IS a possibility that the association finds no records.  So, yes, the code should defend against this possibility.